src/unify/backing_vec.rs

```diff
@@ -17,6 +17,8 @@ pub trait UnificationStore: ops::Index<usize, Output = VarValue<Key<Self>>> + Cl
     type Value: UnifyValue;
     type Snapshot;
 
+    // FIXME(eddyb) (not backwards-compat) this should
+    // use a `Default` bound on the trait instead.
```

Comment:
You might as well change this to use `Default` if you want. We can always issue a new major version.